ddtrace/tracer/option.go

```diff
@@ -829,3 +829,42 @@ func StackFrames(n, skip uint) FinishOption {
 		cfg.SkipStackFrames = skip
 	}
 }
+
+// UserTagOption represents a function that can be provided as a parameter to SetUser.
+// This is used to normalize user tags key/value pairs.
+type UserTagOption func() (key string, val interface{})
+
+// WithUserEmailTag emits the normalized key/value UserTagOption holding user email information.
```

Comment:
```suggestion
// WithUserEmailTag sets email of the authenticated user.
```
Can you apply a similar documentation to your other functions? 🙏
No need to say anything on how it does things, just what it does :-)

Reply:
Setting the e-mail is not really what it does though, the returned closure is (considering the change of implementation you suggested), i.e simply calling the function does nothing, so I find this comment to be misleading.
What it does is emit the option that will set the e-mail, so how about something along `emits the option that enables tracking the email of the authenticated user`. WDYT?

Reply:
Indeed, you can adapt 👍